SPEC.md

```diff
@@ -700,8 +702,8 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest), conformin
     },
     "dependencies": [
         {
-            "app": "example.com/reduce-worker-base",
-            "imageID": "sha512-...",
+            "name": "example.com/reduce-worker-base",
+            "id": "sha512-...",
```

Comment:
perhaps this ID need a more generic name. It is defined as [Image ID](https://github.com/appc/spec/blob/master/SPEC.md#image-id), but could likely be CAS ID or similar